### PR TITLE
Rewrite README intro

### PR DESCRIPTION
**Pull request by pmorie**
Rewrite the README intro and fix a couple other places where federation is likely better written as Kubefed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,28 +4,25 @@
...
-on the API concepts laid down in the [brainstorming
-doc](https://docs.google.com/document/d/159cQGlfgXo6O4WxXyWzjZiPoIuiHVl933B43xhmqPEE/edit#)
-and further refined in the [architecture
-doc](https://docs.google.com/document/d/1ihWETo-zE8U_QNuzw5ECxOWX0Df_2BVfO3lC4OesKRQ/edit#).
-Access to both documents is available to members of the
-[kubernetes-sig-multicluster google
-group](https://groups.google.com/forum/#!forum/kubernetes-sig-multicluster).
+# Kubernetes Cluster Federation (Kubefed)
+
+Kubernetes Cluster Federation (Kubefed for short) allows you to propagate
```
Comment (marun) [inline README.md]: s/propagate resources/configure/

File: README.md [hunk 2]
```diff
@@ -4,28 +4,25 @@
...
+resources to multiple Kubernetes clusters from a single set of APIs in a hosting
+cluster. Kubefed aims to provide essential mechanisms for expressing which
```
Comment (marun) [inline README.md]: s/essential// (by whose measure?)

File: README.md [hunk 3]
```diff
@@ -4,28 +4,25 @@
...
+clusters should be targetted and what resources should go where, with the aim of
```
Comment (marun) [inline README.md]: s/be targeted and what resources should go where/have their configuration managed and what that configuration should be/

File: README.md [hunk 4]
```diff
@@ -4,19 +4,17 @@
...
-on the API concepts laid down in the [brainstorming
-doc](https://docs.google.com/document/d/159cQGlfgXo6O4WxXyWzjZiPoIuiHVl933B43xhmqPEE/edit#)
-and further refined in the [architecture
-doc](https://docs.google.com/document/d/1ihWETo-zE8U_QNuzw5ECxOWX0Df_2BVfO3lC4OesKRQ/edit#).
-Access to both documents is available to members of the
-[kubernetes-sig-multicluster google
-group](https://groups.google.com/forum/#!forum/kubernetes-sig-multicluster).
+
+Kubernetes Cluster Federation (Kubefed for short) allows you to coordinate the
+configuration of multiple Kubernetes clusters from a single set of APIs in a
```
Comment (irfanurrehman) [inline README.md]: I find `configuration` to be a vague, even possibly confusing term to use. For example `kubeconfig` also holds cluster configuration. What are your thoughts on `coordinate deployment and management of kubernetes applications/resources across multiple kubernetes clusters`?

Comment (marun) [reply README.md]: 'configuration' in the context of a cluster is intended to indicate the state of the Kubernetes API.  'resources/applications/etc' are specific terms, but fundamentally kubefed is capable of managing any type of configuration in the API.

Comment (irfanurrehman) [reply README.md]: Ack.

File: README.md (hunk 1 above)
Comment (qinpingli) [reply README.md]: s/Kubefed/KubeFed/

File: README.md [hunk 5]
```diff
@@ -2,21 +2,19 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/kubernetes-sigs/kubefed)](https://goreportcard.com/report/github.com/kubernetes-sigs/kubefed)
 [![Image Repository on Quay](https://quay.io/repository/kubernetes-multicluster/kubefed/status "Image Repository on Quay")](https://quay.io/repository/kubernetes-multicluster/kubefed)
 [![LICENSE](https://img.shields.io/badge/license-apache2.0-green.svg)](https://github.com/kubernetes-sigs/kubefed/blob/master/LICENSE)
-[![Releases](https://img.shields.io/badge/version-v0.0.10-orange.svg)](https://github.com/kubernetes-sigs/kubefed/releases "Kubefed latest release")
-
-# Kubernetes Cluster Federation
```
Comment (marun) [inline README.md]: Was it your intent to remove the document header?

**Pull request merged**